gui/src/components/WorkspaceList.svelte

```diff
@@ -5,8 +5,10 @@
   export let workspaces: WorkspaceDescription[];
 </script>
 
+<button on:click={() => router.push('#/new-project')}> Add new project </button>
```

Comment:
Do we have a LinkButton or Button with href= yet?

Reply:
Not for this styling, but I'm thinking of adding one soon, the main problem is we have similar buttons in different pages with the primary differences being padding/margin/border-radius, so those need to be parameterized.

Reply:
I'm gonna follow up with a separate PR for that, it's not high priority. I think there's already an issue.